src/DssSpell.t.sol

```diff
@@ -205,69 +102,34 @@ contract DssSpellTest is DssSpellTestBase {
-        assertEq(reg.name("PSM-GUSD-A"), "Gemini dollar");
-        assertEq(reg.symbol("PSM-GUSD-A"), "GUSD");
+        assertEq(reg.pos("GUNIV3DAIUSDC2-A"), 46);
+        assertEq(reg.join("GUNIV3DAIUSDC2-A"), addr.addr("MCD_JOIN_GUNIV3DAIUSDC2_A"));
+        assertEq(reg.gem("GUNIV3DAIUSDC2-A"), addr.addr("GUNIV3DAIUSDC2"));
+        assertEq(reg.dec("GUNIV3DAIUSDC2-A"), DSTokenAbstract(addr.addr("GUNIV3DAIUSDC2")).decimals());
+        assertEq(reg.class("GUNIV3DAIUSDC2-A"), 1);
+        assertEq(reg.pip("GUNIV3DAIUSDC2-A"), addr.addr("PIP_GUNIV3DAIUSDC2"));
+        assertEq(reg.xlip("GUNIV3DAIUSDC2-A"), addr.addr("MCD_CLIP_GUNIV3DAIUSDC2_A"));
+        assertEq(reg.name("GUNIV3DAIUSDC2-A"), "Gelato Uniswap DAI/USDC LP");
+        assertEq(reg.symbol("GUNIV3DAIUSDC2-A"), "G-UNI");
     }
 
-    function testDaiVests() public {
-        uint256 lastId = vestDai.ids();
-
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        // General sanity checks
-        // Confirm all new dai vests are under the upper limit of 2M / year
-        // Manually specify special exceptions
-        for(uint256 i = lastId + 1; i <= vestDai.ids(); i++) {
-            assertTrue(vestDai.usr(i) != address(0));
-            assertGt(vestDai.bgn(i), block.timestamp - 90 days);       // Start time is above ~3 months ago
-            assertEq(vestDai.clf(i), vestDai.bgn(i));
-            assertEq(vestDai.mgr(i), address(0));
-            assertEq(vestDai.res(i), 1);
-            assertEq(vestDai.rxd(i), 0);
-
-            uint256 rate = vestDai.tot(i) / (vestDai.fin(i) - vestDai.bgn(i));       // DAI / sec
-            assertLt(rate, 2_000_000 * WAD / 365 days);
-        }
-
-        // Verify individual payments
-        checkDaiVest(++lastId, RWF_WALLET, JAN_01_2022, DEC_31_2022, 1_860_000);
-        checkDaiVest(++lastId, COM_WALLET, DEC_01_2021, DEC_31_2021, 12_242);
-        checkDaiVest(++lastId, COM_WALLET, JAN_01_2022, JUN_30_2022, 257_500);
-        checkDaiVest(++lastId, SAS_WALLET, DEC_01_2021, NOV_30_2022, 1_130_393);
-        checkDaiVest(++lastId, IS_WALLET, DEC_01_2021, AUG_01_2022, 366_563);
-        checkDaiVest(++lastId, MKT_WALLET, DEC_01_2021, APR_30_2022, 424_944);
-        checkDaiVest(++lastId, DECO_WALLET, DEC_01_2021, SEP_01_2024, 5_121_875);
-    }
 
     function testOneTimePaymentDistributions() public {
-        uint256 prevSin      = vat.sin(address(vow));
-        uint256 prevDaiSas   = dai.balanceOf(SAS_WALLET);
-        uint256 prevDaiIs    = dai.balanceOf(IS_WALLET);
-        uint256 prevDaiDeco  = dai.balanceOf(DECO_WALLET);
-
-        uint256 amountSas    = 245_738;
-        uint256 amountIs     = 195_443;
-        uint256 amountDeco   = 465_625;
-        uint256 amountTotal  = amountSas + amountIs + amountDeco;
+        uint256 prevSin              = vat.sin(address(vow));
+        uint256 prevDaiCom           = dai.balanceOf(COM_WALLET);
```

Comment:
These addresses have not been defined in the test.

Reply:
Should be on line 9-14